content/teams/Z3Nspyder/index.md

```diff
@@ -1,133 +1,149 @@
+
+Once an app is flagged, the user can determine whether to whitelist and continue to use the app or can choose to delete it.
+
+The unique feature of B2P is that it runs entirely from the user's side which makes it run quicker than server side applications. B2P is built on and for android devices. It extracts the category of the app from the Google Playstore and thus would work only on applications downloaded directly from the Google Playstore.
+
+
+Conclusion :
+	As privacy is the most important thing in this 21th century, our app provides complete protection to your personal data.You will get an awareness of the various permissions of the apps you install. B2P is very much user friendly and easy to use type with many customizable options. Currently it is a basic prototype but after this hackathon many interesting features will be added. 
+	
+
+
+_**Tech-Stack-Used**_ :   Android, Googl Firebase, Java
+
+_**GitHub-Link**_ :   https://github.com/Z3nSpyder/B2p
+
+_**Hosted-Link**_ :    N/A
 
 ---
 
 # Member Details
 
-_**No of members**_ : 
+_**No of members**_ : 6
 
 Fill Team Details for only the valid no of members in your team and delete the leftover Template...
 
 ## Leader Details
 
-_**Name**_ :
+_**Name**_ : Sai Ganesh S
 
-_**University-Name**_ : 
+_**University-Name**_ : Anna University, MIT campus
 
-_**country**_ :
+_**country**_ : India
  
-_**Technical-Skills**_ :
+_**Technical-Skills**_ :c/c++, python, js, cyber threat intelligence, basic ML
 
-_**Specialization**_ :
+_**Specialization**_ :B.E Computer Science And Engineering
 
-_**GitHub-ID**_ :  
+_**GitHub-ID**_ : www.github.com/appleo11official 
 
 ---
 
 ## Member 2 Details
 
-_**Name**_ :
+_**Name**_ : Haniel Edward Jacob
 
-_**University-Name**_ : 
+_**University-Name**_ : St. Joseph's College of Engineering
 
-_**country**_ :
+_**country**_ : India
  
-_**Technical-Skills**_ :
+_**Technical-Skills**_ :C/C++,  Python, Java, Web development, Android development, Machine Learning
 
-_**Specialization**_ :
+_**Specialization**_ :B.E Computer Science And Engineering
 
-_**GitHub-ID**_ :   
+_**GitHub-ID**_ : www.github.com/hanieljacob  
 
 ---
 
 ## Member 3 Details
 
-_**Name**_ :
+_**Name**_ : Joshua Pradeep
 
-_**University-Name**_ : 
+_**University-Name**_ :  St. Joseph's College Of Engineering 
 
-_**country**_ :
+_**country**_ : India 
  
-_**Technical-Skills**_ :
+_**Technical-Skills**_ : C/C++   python, java, basic ML, HTML, CSS
 
-_**Specialization**_ :
+_**Specialization**_ : B.E Computer Science And Engineering
 
-_**GitHub-ID**_ :   
+_**GitHub-ID**_ :  www.github.com/jpjosh4  
```

Comment:
Small changes requested :  remove www and add https://github.com/userID

And the PR will be perfect

Reply:
Done